KS-2-4 Buff

CasTeTRU opened the pull request.

CasTeTRU committed:
KS-2-4 Buff

CasTeTRU committed:
KS-2-4 Buff
Output amount of crystal and iron ore under the buff

ievetrovdev commented on src/main/kotlin/lesson_2/lesson2_task4.kt:
```diff
@@ -0,0 +1,9 @@
+fun main() {
+    val crystalOre: Short = 7
+    val ironOre:Short = 11
+    val buffAmount = 0.2
```
Для возможности гибкой работы с кодом, следует инициализировать согласно ТЗ проценты целым числом 20. И уже потом производить вычисление процентов. Таким образом, если нужен будет другой процент, достаточно будет поменять значение в одном месте.

CasTeTRU committed:
KS-2-4-Buff
Проценты возвращены к числовой записи
Добавлен делитель для вычисления процента

ievetrovdev reviewed (approved):
Теперь все четко!
